### PR TITLE
Fix: try catch console errors

### PR DESCRIPTION
**Pull request by RichardIrala**

### CODE REVIEW

File: atoms/XerialWallet/XerialWallet.jsx [hunk 1]
```diff
@@ -153,7 +153,7 @@ const XerialWallet = ({ XerialWalletViewmodel }) => {
       await navigator.clipboard.writeText(text);
       console.log("Content copied to clipboard");
     } catch (err) {
-      console.error("Failed to copy: ", err);
+      console.error("Error to copy text. Reason: ", err.message);
```
Comment (luislucena16) [inline atoms/XerialWallet/XerialWallet.jsx]: without the comma (`,`) for all would be fine...

File: functions/web3/web3.js [hunk 2]
```diff
@@ -74,7 +74,7 @@ web3Functions.secondaryMarketPurchase = async function ({ marketplaceNftId }) {
     const purchaseTransactionWaited = await purchaseTransaction.wait();
     console.log(purchaseTransactionWaited);
   } catch (error) {
-    console.error("Error to purchase NFT in Secondary Market");
+    console.error("Error to purchase NFT in Secondary Market. Reason: ", error.message);
```
Comment (luislucena16) [inline functions/web3/web3.js]: how would you set the logs with function names or with descriptive names for you, define one... example:
error structure:
1. function with message + error
2. message + error

choose the best one for your development!

**Review by luislucena16 — CHANGES_REQUESTED** [review]

File: functions/web3/web3.js (hunk 2 above)
Comment (RichardIrala) [reply functions/web3/web3.js]: Done

File: atoms/XerialWallet/XerialWallet.jsx (hunk 1 above)
Comment (RichardIrala) [reply atoms/XerialWallet/XerialWallet.jsx]: Done

**Review by luislucena16 — APPROVED** [review]

**Pull request merged**